extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -129,10 +133,20 @@ export class SubscriptionService {
         /** @type {!PageConfig} */
         this.pageConfig_ = promiseValues[1];
       });
+      this.maybeDecryptDocument_();
```

Comment:
I understand we can no longer start this right away since we don't know which service will be used?

Reply:
moved